addons/account/models/account_move.py

```diff
@@ -2552,11 +2552,16 @@ def button_draft(self):
     def button_cancel(self):
         self.write({'auto_post': False, 'state': 'cancel'})
 
+    def _can_send_invoice(self):
+        return self.user_has_groups('account.group_account_invoice')
+
     def action_invoice_sent(self):
         """ Open a window to compose an email, with the edi invoice template
             message loaded by default
         """
         self.ensure_one()
+        if not self._can_send_invoice():
```

Comment:
Just checking that the user has write access on `self` is enough and doesn't require adding that new method nor the new string. The standard error of `check_access_rights` should be enough.